odoo/tools/misc.py

```diff
@@ -26,7 +26,7 @@
 import werkzeug.utils
 import zipfile
 from collections import defaultdict, Iterable, Mapping, MutableMapping, MutableSet, OrderedDict
-from itertools import islice, groupby as itergroupby, repeat
+from itertools import islice, groupby as itergroupby, repeat, chain
```

Comment:
not used, in fact

Reply:
Yeah I was using chain.from_iterable in the original iterate but it didn't work right, forgot to remove the import when I changed the implementation.

Should probably rename iterate as well, though not sure what name to use there. iterate_struct? iterate_builtin?